Errors in build

```
rpmbuild -ba incus.spec               
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.13883
+ umask 022
+ cd /home/builder/rpmbuild/BUILD
+ '[' 1 -eq 1 ']'
+ '[' 1 -eq 1 ']'
+ '[' 1 -eq 1 ']'
+ cd /home/builder/rpmbuild/BUILD
+ rm -rf incus-incus-0.1
+ /bin/gzip -dc /home/builder/rpmbuild/SOURCES/incus-0.1.tar.gz
+ /usr/bin/tar -xf -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd incus-incus-0.1
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.13883
+ umask 022
+ cd /home/builder/rpmbuild/BUILD
+ cd incus-incus-0.1
+ '[' 1 -eq 1 ']'
+ '[' 1 -eq 1 ']'
+ export CC=/opt/llvm16/bin/clang
+ CC=/opt/llvm16/bin/clang
+ export CXX=/opt/llvm16/bin/clang++
+ CXX=/opt/llvm16/bin/clang++
+ export CPP=/opt/llvm16/bin/clang-cpp
+ CPP=/opt/llvm16/bin/clang-cpp
+ export PATH=/opt/llvm16/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ PATH=/opt/llvm16/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ export GOPROXY=direct
+ GOPROXY=direct
+ /usr/bin/make -j8 build
make: command: Command not found
make: command: Command not found
make: command: Command not found
CC="/opt/llvm16/bin/clang" CGO_LDFLAGS_ALLOW="(-Wl,-wrap,pthread_create)|(-Wl,-z,now)" go install -v -tags "libsqlite3"  ./...
github.com/lxc/incus/internal/server/seccomp
# github.com/lxc/incus/internal/server/seccomp
internal/server/seccomp/seccomp.go:342:23: error: 'aligned' attribute ignored when parsing type [-Werror,-Wignored-attributes]
internal/server/seccomp/seccomp.go:254:25: note: expanded from macro 'ptr_to_u64'
/usr/include/linux/types.h:43:44: note: expanded from macro '__aligned_u64'
internal/server/seccomp/seccomp.go:347:21: error: 'aligned' attribute ignored when parsing type [-Werror,-Wignored-attributes]
internal/server/seccomp/seccomp.go:254:25: note: expanded from macro 'ptr_to_u64'
/usr/include/linux/types.h:43:44: note: expanded from macro '__aligned_u64'
internal/server/seccomp/seccomp.go:348:22: error: 'aligned' attribute ignored when parsing type [-Werror,-Wignored-attributes]
internal/server/seccomp/seccomp.go:254:25: note: expanded from macro 'ptr_to_u64'
/usr/include/linux/types.h:43:44: note: expanded from macro '__aligned_u64'
make: *** [Makefile:35: build] Error 1
error: Bad exit status from /var/tmp/rpm-tmp.13883 (%build)

```

```
%global        optflags %(echo %{optflags} | sed -e 's/-ffat-lto-objects//g' -e 's/-fvar-tracking-assignments//g' -e 's/-Wa,--compress-debug-sections//g')
%define         debug %nil
%define         debug_-package %nil

Name:           incus
Version:        0.1
Release:        1
Summary:        Powerful system container and virtual machine manager
URL:            https://linuxcontainers.org/incus
Source0:        https://github.com/lxc/incus/archive/%{name}-%{version}.tar.gz
License:        ASL 2.0
Group:          Development/Other
BuildRequires:  go
BuildRequires:  git-core
BuildRequires:  upx
BuildRequires:  pkgconfig(libuv) >= 1.8
BuildRequires:  pkgconfig(sqlite)
BuildRequires:  pkgconfig(cowsql)
BuildRequires:  pkgconfig(raft) >= 0.17.5
BuildRequires:  pkgconfig(libcap)
BuildRequires:  pkgconfig(lxc)
BuildRequires:  pkgconfig(libacl)
BuildRequires:  pkgconfig(libudev)
BuildRequires:  pkgconfig(libseccomp)
BuildRequires:  llvm16

%description
...

%prep
%setup -qn %{name}-%{name}-%{version}

%build

export CC=/opt/llvm16/bin/clang
export CXX=/opt/llvm16/bin/clang++
export CPP=/opt/llvm16/bin/clang-cpp

export PATH=/opt/llvm16/bin:$PATH

export GOPROXY=direct

%make build
```

```
lsb_release -a
LSB Version:    *
Distributor ID: RosaDesktopFresh
Description:    ROSA Desktop Fresh R11.1
Release:        2016.1
Codename:       Fresh
```

go version  
go version go1.21.1 linux/amd64







<!--
Github issues are used for bug reports. For support questions, please use [our forum](https://discuss.linuxcontainers.org/).

Please fill the template below as it will greatly help us track down your issue and reproduce it on our side.
Feel free to remove anything which doesn't apply to you and add more information where it makes sense.
-->

# Required information

 * Distribution:
 * Distribution version:
 * The output of "inc info" or if that fails:
   * Kernel version:
   * LXC version:
   * Incus version:
   * Storage backend in use:

# Issue description

A brief description of the problem. Should include what you were
attempting to do, what you did, what happened and what you expected to
see happen.

# Steps to reproduce

 1. Step one
 2. Step two
 3. Step three

# Information to attach

 - [ ] Any relevant kernel output (`dmesg`)
 - [ ] Container log (`inc info NAME --show-log`)
 - [ ] Container configuration (`inc config show NAME --expanded`)
 - [ ] Main daemon log (at /var/log/incus/incus.log)
 - [ ] Output of the client with --debug
 - [ ] Output of the daemon with --debug (alternatively output of `inc monitor` while reproducing the issue)

--- a/internal/server/seccomp/seccomp.go
+++ b/internal/server/seccomp/seccomp.go
@@ -251,7 +251,7 @@ static inline int pidfd_getfd(int pidfd, int fd, int flags)
 	return syscall(__NR_pidfd_getfd, pidfd, fd, flags);
 }
 
-#define ptr_to_u64(p) ((__aligned_u64)((uintptr_t)(p)))
+#define ptr_to_u64(p) ((__u64)((uintptr_t)(p)))
 
 static inline int bpf(int cmd, union bpf_attr *attr, size_t size)
 {

